stories/Atom/Images/Authorimage/authorimage.scss

```diff
@@ -8,6 +8,7 @@
   height: 80px;
   overflow: hidden;
   width: 80px;
+  min-width: 80px;
```

Comment:
@Shamimkhan321 No point using both width and min-width if they are fixed to `80px` instead use width on `.author-label` div. Eg. `width: calc( 100% - 80px)` for small and `width: calc( 100% - `180px)` for large on the `.author-label` container. Also if possible use global variable for `small: 80px` and `large: 180px` on image.